src/format.rs

```diff
@@ -189,17 +189,37 @@ impl PayloadParams {
         if c0 == c1 {
             return Some(100);
         } else {
-            // TODO: fuzzy matching.
-        }
+            if c0.codec != c1.codec || (c0.codec == Codec::Unknown && c1.codec == Codec::Unknown) {
+                // Codecs must match
+                return None;
+            }
+
+            if c0.clock_rate != c1.clock_rate {
+                // Clock rates must match
+                return None;
+            }
+
+            if c0.channels != c1.channels {
+                // Channels must match
+                return None;
+            }
+
+            if c0.codec.is_audio() {
+                // Accept whatever FEC and p-time setting the remote proposes
```

Comment:
This is very liberal i.e. we cannot force FEC or a specific p-time. Do we want some way to determine hard requirements, .e.g `FEC` must be enabled